sql/src/main/java/io/crate/analyze/SetStatementAnalyzer.java

```diff
@@ -83,6 +83,24 @@ public static ResetAnalyzedStatement analyze(ResetStatement node) {
         return new ResetAnalyzedStatement(settingsToRemove);
     }
 
+    /**
+     * Rewrites old settings to new settings
+     * This will be removed in CrateDB v1.1
+     */
+    @Deprecated
+    private static Assignment rewriteOldSettings(Assignment assignment) {
+        // parser does not allow using the parameter expressions as setting names in the SET statements,
+        // therefore it is fine to convert the expression to string here.
+        String settingName = ExpressionToStringVisitor.convert(assignment.columnName(), Row.EMPTY);
```

Comment:
why not just act on the assignment.columnName() instead of do this forth and back converting?